circle.yml

```diff
@@ -0,0 +1,82 @@
+version: 2
+workflows:
+  version: 2
+  linux-builds:
+    jobs:
+      - build-x86_64
+      - build-i686
+
+jobs:
+  build-x86_64:
+    docker:
+#      - image: ubuntu:14.04
+      - image: circleci/python:2.7
```

Comment:
elliot's images also do 32 bit builds in a 64 bit image.

ci and buildbots for binaries serve different purposes, so no. binaries need to be built with an old glibc and new compiler, but ci should test a more common configuration - make sure we don't start using features that only work in super new gcc until llvm requires us to